src/layer/index.ts

```diff
@@ -33,7 +33,11 @@ export { Target, target } from './target';
 export { Transpose, transpose } from './transpose';
 export { Zeros, zeros } from './zeros';
 
-export const layerTypes = {
+interface layerTypesMap {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  [key: string]: any;
```

Comment:
There should be a way to avoid `any` type

Reply:
I think I'll deal with https://github.com/BrainJS/brain.js/pull/611#discussion_r499437605 first and then this should be pretty easy.